recipes/tcsh/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "tcsh" %}
+{% set version = "6.20.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tcsh-org/tcsh.git
+  git_tag: TCSH6_20_00
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - ncurses
+    - gcc
```

Comment:
This should be dropped. We include a system compiler that should be used instead (if at all possible).